maas-schemas/schemas/maas-backend/booking-virtual-create/response.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$id": "http://maasglobal.com/maas-backend/booking-virtual-create/response.json",
+  "description": "Response schema for booking-virtual-create",
+  "type": "array",
```

Comment:
I think for responses, we usually return an object, and then have a property, e.g. `invoices`. This makes it a bit easier if we want to extend it?